docs/pages/site.css

```
@@ -117,7 +123,19 @@
+  /* Три системы одной тихой строкой: значок и слово, без рамок. Многие
+     инструменты для 1С живут только на Windows, поэтому это стоит сказать
+     сразу, но не громче кнопок. На десктопе строка стоит в ряду с кнопками
+     и закрывает пустоту справа, на телефоне уходит под кнопки по центру. */
+  .platforms { display: flex; flex-wrap: wrap; gap: 6px 22px; margin: 0 0 0 10px; font-size: 13px; color: var(--muted); }
+  /* Число сценариев живёт по тем же правилам: справа от кнопок, на телефоне
+     под ними по центру. Текстом, не баблом: бабл рядом с кнопками читается
+     как ещё одна кнопка. */
+  .cta-note { margin: 0 0 0 10px; font-size: 14px; color: var(--muted); }
+  .cta-note b { font-size: 17px; font-weight: 680; color: var(--ink); margin-right: 2px; }
+  .os { display: inline-flex; align-items: center; gap: 7px; }
+  .os svg { width: 15px; height: 15px; fill: currentColor; opacity: 0.85; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)